app/src/main/java/com/android/sample/ui/composables/ExerciseCard.kt

```diff
@@ -67,11 +70,15 @@ fun ExerciseCard(exercise: Exercise) {
  * @param detail the [ExerciseDetail] data to display in the card.
  */
 @Composable
-fun ExerciseDetailCard(detail: ExerciseDetail) {
+fun ExerciseDetailCard(detail: ExerciseDetail, onClick: () -> Unit) {
   Card(
       shape = RoundedCornerShape(12.dp),
       colors = CardDefaults.cardColors(containerColor = Blue), // Darker blue color
-      modifier = Modifier.padding(start = 8.dp).wrapContentSize()) {
+      modifier =
+          Modifier.padding(start = 8.dp).testTag("detailCard").wrapContentSize().clickable {
+            onClick()
+          }) {
+        Text(modifier = Modifier.testTag("etailCard"), text = "Details", color = Color.White)
```

Comment:
You should use our color defined in Color.kt

And also this test tag seems useless